### PR TITLE
Makes the alien and the AI fake deathgasp make more sense

### PR DESCRIPTION
**Pull request by ghost**
(untested because my laptop's hard drive gave up on me)  
Because MEH IMMURSHON and stuff.  Can understand if people think this is a horrible Idea I just don't see how the whole faking blood and sparks thing makes sense
(I'm very aware that this would mean players could tell if aliens/AI's are faking their death however I don't think it it'll be that much of an issue) 

(I swear if the deathgasp emote is also used when you actually die and are not just faking it I will scream) 

Hopefully I'm doing this right.

(Hey can you tell I like brackets)

:cl: name
tweak: Made the AI and alien fake deathgasps make more sense
/:cl:

### CODE REVIEW

**Comment by PepperPrepper** [comment]
Ignore the commits from master. I'm dumb and have no idea how to remove them (it was just me updating my branch)

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -74,8 +74,8 @@
 	key_third_person = "deathgasps"
 	message = "seizes up and falls limp, their eyes dead and lifeless..."
 	message_robot = "shudders violently for a moment before falling still, its eyes slowly darkening."
-	message_AI = "lets out a flurry of sparks, its screen flickering as its systems slowly halt."
-	message_alien = "lets out a waning guttural screech, green blood bubbling from its maw..."
+	message_AI = "memory dumps, its screen flickering as its systems slowly halt."
+	message_alien = "lets out a waning guttural screech, And collapses onto the floor..."
```
Comment (spessbandit) [inline code/modules/mob/living/emote.dm]: why is this capitalized

Comment (PepperPrepper) [reply code/modules/mob/living/emote.dm]: Because my typing skills are lackluster

**Comment by Tharcoonvagh** [comment]
do this with more "races"

**Comment by ATH1909** [comment]
It might just be me, but "memory dumps" is kind of a weird/an awkward phrase. Maybe change that to something else?

**Comment by PepperPrepper** [comment]
@ATH1909 Do you have any ideas that I could replace "memory dumps" with?

I completely agree with you, it sounds really weird but I couldn't come up with anything else.

**Comment by Tharcoonvagh** [comment]
"alarms"?

**Comment by PepperPrepper** [comment]
@ATH1909 
I'll take your suggestion and go with screeches

**Pull request merged**